# This message is to send standard unitary actions to the OM actuator
Header header #for timestamp

int16 actionID  #to identify each unitary action
string result   # string defining if the action SUCCEEDED or FAILED
string log      # log information about the result of the action
